### PR TITLE
Syntax and CSS stylesheets

### LINKED ISSUE #39: Delete commented CSS, improving syntax
Need to delete commented CSS from SideDrawer.css, add a 0 before any decimal on SideDrawer.css and Toolbar.css. From SideDrawer.jsx delete spacing in </div >

### PR DESCRIPTION
**Pull request by amgarz0n**
Closes #39

### CODE REVIEW

File: client/src/pages/WelcomePage.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import '../index.css';
-import { BrowserRouter as Router, Link, Route, Switch } from 'react-router-dom';
+import { BrowserRouter as Link } from 'react-router-dom';
```
Comment (Rterrell25) [inline client/src/pages/WelcomePage.jsx:3]: should be 
`import {Link} from react-router-dom`

File: client/src/App.jsx [hunk 2]
```diff
@@ -17,13 +14,10 @@ import MentalHygiene from './pages/MentalHygiene';
...
 import TherapistSearch from './pages/TherapistSearch';
 import MyTherapist from './pages/MyTherapist';
-
-
-
 import Toolbar from './components/Toolbar/Toolbar';
 import TherapistProfile from './pages/TherapistProfile/TherapistProfile';
 import SideDrawer from './components/SideDrawer/SideDrawer';
-import { AppContextProvider } from './context/AppContext';
+// import { AppContextProvider } from './context/AppContext';
```
Comment (Rterrell25) [inline client/src/App.jsx:20]: Are you no longer using AppContext provider throughout your application?

File: client/src/pages/WelcomePage.jsx (hunk 1 above)
Comment (amgarz0n) [reply client/src/pages/WelcomePage.jsx]: Hey Robert, just changed line 3 from {BrowserRouter} to {Link}, pushed that change to this branch

File: client/src/App.jsx (hunk 2 above)
Comment (amgarz0n) [reply client/src/App.jsx:20]: We want to use it, but we have not done it yet. Since it was giving errors, I commented it out.

**Pull request merged**